backend/src/services/yelp_service.py

```
@@ -148,7 +159,9 @@ async def _make_request(self, endpoint: str, params: Dict[str, Any] = None) -> D
                         return await response.json()
                     else:
                         error_data = await response.json()
-                        error_msg = error_data.get("error", {}).get("description", "Unknown error")
+                        error_msg = error_data.get("error", {}).get(
+                            "description", "Unknown error"
+                        )
                         raise ServiceError(f"Yelp API error: {error_msg}")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)